service_mesh/v2x/installing-ossm.adoc

```diff
@@ -13,6 +13,11 @@ To install {SMProductName}, first install the required Operators on {product-tit
 This basic installation is configured based on the default OpenShift settings and is not designed for production use.  Use this default installation to verify your installation, and then configure your service mesh for your specific environment.
 ====
 
+[NOTE]
```

Comment:
This is is a duplication of the note directly preceding it. You could remove this one so that there is only one instance of the note in this introduction section.

Was this intended to add the following, as per another example in this PR?:

> **Note**: The Service Mesh documentation uses istio-system as the example project, but you can deploy the service mesh to any project.